### DISCUSSION TITLE
posts/026-jp-speeding/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#12] [Announcements]
# posts/026-jp-speeding/

牠們遵循抓超速的《最佳實踐》，但是被俺一眼識破😁👎🏻

**Comment by PermeationLoop** [comment]
之後還有一篇，來扒一扒日本的交通標誌；
能做出那種不合理設計的多半腦袋都被門夾過，俺先放一張圖預告一下：

<img width="224" height="340" alt="到底是甚麼意思？" src="https://github.com/user-attachments/assets/e98ad274-0c7e-4a80-a407-05fe0235d966">

**Comment by PermeationLoop** [comment]
**近期日本條子見聞更新**

1. 昨天從街上回家的路上，看到一輛正準備開進⛔️的車被條子當場攔下；可能是因爲這個條子只是一般通過巡邏，警告了兩句就直接放他走了，，，

2. 今天從超市回家的路上，走到路口看到一輛條子麪包車亮着『事故處理中』的牌子停在路邊，而那裏並沒有任何事故的痕跡，所以很明顯就是在蹲違章的（蹲不禮讓行人的）🤮👎🏻
